docs/organizations/managing-security-and-risk.md

```diff
@@ -161,7 +161,23 @@ Codacy closes a finding when it detects that the associated Jira issue is marked
 !!! note
     Penetration testing is available upon request and is provided by a third-party partner. See [how to request penetration testing for your organization](https://go.codacy.com/pen-testing-product).
 
-Codacy opens a new finding when penetration testing results are published by our partner.
+Codacy opens a new finding when penetration testing results are published by our partner.<!-- TODO TAROT-2669: clarify when these are closed -->
+
+## Finding severities and deadlines {: id="item-severities-and-deadlines"}
```

Comment:
Note: Feel free to ignore this section when reviewing. I moved it to ensure consistency with the UI.